detox/jest.detox.startup.js

```diff
@@ -2,11 +2,30 @@ const { by, cleanup, init, device, element, waitFor } = require("detox");
 const adapter = require("detox/runners/jest/adapter");
 const specReporter = require("detox/runners/jest/specReporter");
 const config = require("./detox.config");
+const { toMatchImageSnapshot } = require("jest-image-snapshot");
+const path = require("path");
 
 jest.setTimeout(300000);
 jasmine.getEnv().addReporter(adapter);
 jasmine.getEnv().addReporter(specReporter);
 
+expect.extend({
+    toMatchImageSnapshot(screenshot, options = {}) {
+        const { currentTestName } = this;
+        const platform = device.getPlatform();
+        const customSnapshotsDir = path.join(path.resolve("./"), "image-snapshots", platform);
```

Comment:
`join` and `resolve` can be destructured from the import directly so you don't have to prefix it with `path.`.

const { join, resolve } = require("path")

Reply:
Good one, made the changes 👍